package.json

```diff
@@ -3,7 +3,8 @@
   "scripts": {
     "lint:js": "eslint .",
     "lint:ts": "tsc",
-    "test": "UNIVERSE_I18N_LOCALES=en-US meteor test-packages --port 8080 --driver-package meteortesting:mocha ./"
+    "test": "UNIVERSE_I18N_LOCALES=en-US meteor test-packages --port 8080 --driver-package meteortesting:mocha ./",
+    "test:once": "./tests/tests.sh"
```

Comment:
The latter does far more than just "test once" when compared to "test". Maybe we could name it better? Or simply get rid of `test`?